src/inmanta/parser/chache.py

```diff
@@ -0,0 +1,100 @@
+"""
+    Copyright 2021 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import logging
+import os
+from typing import List, Optional
+
+from inmanta.ast import Namespace
+from inmanta.ast.statements import Statement
+from inmanta.compiler.config import feature_compiler_cache
+from inmanta.parser.pickle import ASTPickler, ASTUnpickler
+from inmanta.util import get_compiler_version
+
+LOGGER = logging.getLogger(__name__)
+
+
+class CacheManager:
+    def __init__(self) -> None:
+        self.hits = 0
+        self.misses = 0
+        self.failures = 0
+
+    def get_file_name(self, filename: str) -> Optional[str]:
```

Comment:
I think a docstring would be helpful here. The method name does not state sufficiently clearly what it does to go without.

Reply:
thx, added some comments